README.md

```diff
@@ -0,0 +1,17 @@
+
+Teckers Android App
+
+Teckers is a school dedicated to girls and boys between nine and nineteen years old, to help them develop and strengthen the skills and knowledge necessary to create technological solutions to social problems in their community.
+
+The android app of "teckes" plans to facilitate the handling and communication between students and teckers in a faster and more precise way.
+
+It will work giving different types of tools, such as messaging and attendance control management in students.
+
+We work with the first version of the app, currently under development.
+
+We are working based on the latest version of Android Studio, implementing Kotlin for development.

```

Comment:
We have to be very specific with versions because this sentence might not be true if the latest version of AS changes and we don't update.